mmdet/datasets/coco.py

```diff
@@ -60,6 +69,26 @@ def _filter_imgs(self, min_size=32):
                 valid_inds.append(i)
         return valid_inds
 
+    def get_imgs_by_cat(self, imgIds=[], catIds=[]):
```

Comment:
- `get_subset_by_classes()` looks better.
- `img_ids`, `class_ids`
- What is `img_ids` for? It seems to be unused.
- Use tuple or None as default values.
- Use the standard docsting style.